cmd/run.go

```diff
@@ -71,11 +74,16 @@ var runCmd = &cobra.Command{
 			MaxMarkerRetries:  maxMarkerRetries,
 			MaxMarkerLogLines: maxMarkerLogLines,
 		}, out)
+
 		if err != nil {
 			log.Fatal().Err(err)
+			return err
 		}
-
-		os.Exit(currentRun.Stats.TotalFailed())
+		if currentRun.Stats.TotalFailed() > 0 {
+			errText := fmt.Errorf("failed %d tests", currentRun.Stats.TotalFailed())
```

Comment:
`Errorf` returns `error`, no need for `errors.New(...)`.

Reply:
Fixed.